core/db/migrate/20150314013438_add_missing_indexes.rb

```diff
@@ -1,27 +1,25 @@
 class AddMissingIndexes < ActiveRecord::Migration
   def change
-    add_index   :spree_promotion_rules_users, 
-                        [:user_id, :promotion_rule_id], 
+    add_index   :spree_promotion_rules_users,
+                        [:user_id, :promotion_rule_id],
                         name: 'index_promotion_rules_users_on_user_id_and_promotion_rule_id'
-    add_index   :spree_products_promotion_rules, 
-                        [:promotion_rule_id, :product_id], 
+    add_index   :spree_products_promotion_rules,
+                        [:promotion_rule_id, :product_id],
                         name: 'index_products_promotion_rules_on_promotion_rule_and_product'
     add_index :spree_orders, :canceler_id
     add_index :spree_orders, :store_id
     add_index :spree_orders_promotions, [:promotion_id, :order_id]
     add_index :spree_properties_prototypes, :prototype_id
-    add_index   :spree_properties_prototypes, 
-                        [:prototype_id, :property_id], 
+    add_index   :spree_properties_prototypes,
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.